### ISSUE TITLE
autocomplete error on select

### ISSUE THREAD

**Issue by alexey2baranov** [museui/muse-ui#250]
Hi! i  write very simple test component 

```html
<template>
    <div>
            <mu-auto-complete :dataSource="autocomplete_dataSource" />
    </div>
</template>

<script>
    export default {
        name: 'app-view',
        data: function () {
            return {
                autocomplete_dataSource: ["11", "22"]
            }
        },
    }
</script>
```

When I write "1" autocomplete offers me "11". Then I select "11" and get this error into Chrome console 

> vue.common.js?e881:2395 Uncaught TypeError: Cannot read property '_events' of undefined
>     at Vue.$emit (eval at <anonymous> (app.js:683), <anonymous>:2395:17)
>     at eval (eval at <anonymous> (app.js:955), <anonymous>:5:19394)

Where is my mistake?

**Comment by myronliu347** [comment]
Please write demo in  https://jsfiddle.net/

Resources
https://unpkg.com/vue/dist/vue.min.js
https://unpkg.com/muse-ui/dist/muse-ui.js
https://unpkg.com/muse-ui/dist/muse-ui.css

**Comment by alexey2baranov** [comment]
Ok. I wrote this on jsfiddle. why did you close my issue?

https://jsfiddle.net/ypx9ztrx/2/

**Comment by myronliu347** [comment]
This seems to be a bug, I will fix in the next version, Thank you

**Comment by linxiaowu66** [comment]
Hi @alexey2baranov 
  This issue has solved in next version. Thanks a lot

**Comment by myronliu347** [comment]
Fixed in 2.0.0-rc.5, please update
